src/components/molecules/LogoutButton.tsx

```
@@ -0,0 +1,131 @@
+          variant="ghost"
+          loading={loading}
+          onPress={confirmLogout}
+          style={style}
+          accessibilityLabel="Logout"
+          accessibilityHint="Keluar dari aplikasi"
+        >
+          Logout
+        </Button>
+      );
+
+    case 'button':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)